#163745004 Modularize routes

czarjulius opened the pull request:
#### What does this PR do?
It gives my code a good organization. 

#### Description of Task to be completed?
 -removed comments
 -ignored node-modules
 -changed status code
#### How should this be manually tested?
Navigate to https://github.com/czarjulius/Politico/
click on server and view the codes
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
#### Screenshots or gifs (if appropriate)
#### Questions:

czarjulius committed:
bug(fixedMigrations) : installed @babel-node -ensure all fields captured [Delivers #163713875]

czarjulius committed:
feat(createUser): User can sign up [Delivers #163674173]

czarjulius committed:
chore(modularization): separated routes
-removed comments
-ignored node-modules
-changed status code
[Delivers #163745004]

ascii-dev commented:
Kindly update the first two commits to follow the patter of the last commit.

ascii-dev commented on server.js:
```diff
@@ -1,9 +1,9 @@
 import express from 'express';
-import router from './server/routes';
+import router from './server/routes/index';
```
If the file is named `index.js` you don't need to include it here. JS automatically looks for `index.js` if the `path to import` is a folder.

ascii-dev commented on server/controllers/AuthUserController.js:
```diff
@@ -8,7 +8,7 @@ dotenv.config();
   static registerUser(req, res) {
     const {
-      firstName, lastName, otherName, phoneNumber, passportUrl, isAdmin, email, password,
+      firstName, lastName, otherName, phoneNumber, passportUrl, email, password,
```
Any reason for removing `isAdmin`?

ascii-dev commented on server/controllers/OfficeController.js:
```diff
@@ -1,46 +1,53 @@
-    });
+  static getOffice(req, res) {
+    const officeId = req.params.id;
+    const query = `SELECT * from office where id ='${officeId}'`;
```
Any check for the case where the ID does not exist in the database?

ascii-dev commented on server/routes/officeRoute.js:
```diff
@@ -0,0 +1,24 @@
+
+router.get(
+  '/offices/:id',
+  OfficeController.getOffice,
```
You're not validating that I actually passed in an ID. I think you should do that.

ascii-dev commented on server/routes/partyRoute.js:
```diff
@@ -0,0 +1,35 @@
+
+router.get(
+  '/parties/:id',
+  PartyController.getParty,
```
You're not validating that I actually passed in an ID. I think you should do that.

ascii-dev commented on server/routes/partyRoute.js:
```diff
@@ -0,0 +1,35 @@
+
+router.delete(
+  '/parties/:id',
+  PartyController.deleteParty,
```
You're not validating that I actually passed in an ID. I think you should do that.

ascii-dev commented on server/routes/userRoute.js:
```diff
@@ -0,0 +1,12 @@
+  '/auth/signup',
+  userValidator.validateRegistrationEntry,
+  AuthController.registerUser,
+);
```
Kindly add a new line below this line. Thanks

czarjulius committed:
(feedBack): Added a helper class
- ensured my test is not failing
- ensure a nice indentation [Delivers #163745004]
